### DISCUSSION TITLE
Want to use Orbit

### DISCUSSION THREAD

**Discussion by PilauGrice** [ryangjchandler/orbit#147] [Q&A]
Hi i want use orbit but everytime i go to run a user create it just give me thsi error. 

Illuminate\Database\QueryException with message 'SQLSTATE[HY000]: General error: 5 database is locked (SQL: insert into "users" ("name", "email", "updated_at", "created_at") values (cally, admin@admin.com, 2022-10-18 15:46:23, 2022-10-18 15:46:23))'

thanks in advance.

**Comment by PilauGrice** [comment]
Ignore the above post i have now sorted this but no YAML file is being created when the creation has run.

**Reply by ryangjchandler** [reply]
Can you share your model configuration please.

**Reply by PilauGrice** [reply]
`<?php

namespace App\Models;

use Orbit\Concerns\Orbital;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Schema\Blueprint;
use Illuminate\Database\Eloquent\Factories\HasFactory;

class Post extends Model
{
    use HasFactory;

    use Orbital;

    public static $driver = 'yaml';

    public static function schema(Blueprint $table)
    {
        $table->id()->unique();
        $table->string('name');
        $table->string('email');
        $table->timestamp('email_verified_at')->nullable();
        $table->string('password');
        $table->rememberToken();
    }
}
`

**Reply by PilauGrice** [reply]
Hey Man 

I managed to get the User working but now am trying to add a role 


`<?php

namespace App\Models;

use Orbit\Concerns\Orbital;
use Laratrust\Models\LaratrustRole;
use Illuminate\Database\Schema\Blueprint;

class Role extends LaratrustRole
{
    public $guarded = [];


    use Orbital;

	public static function schema(Blueprint $table)
	{
        $table->bigIncrements('id');
        $table->string('name')->unique();
        $table->string('display_name')->nullable();
        $table->string('description')->nullable();
        $table->timestamps();
	}
}
`
that's my model but get this error when i run $Role->save(); within tinker. after doing all the other ones name etc.

Illuminate\Database\QueryException with message 'SQLSTATE[HY000]: General error: 1 no such table: roles (SQL: insert into "roles" ("name", "display_name", "description", "updated_at", "created_at") values (Creative, Creative, Creative Staff, 2022-10-20 11:18:42, 2022-10-20 11:18:42))'

**Reply by ryangjchandler** [reply]
Try clearing orbit cache : `artisan orbit:clear`

I'm not sure how compatible it will be with third party models though, looks like you're using a model from Laratrust. That's not an officially supported use case for the package.

**Reply by PilauGrice** [reply]
Yeah it is a third party thing it created the folders fine but it hasnt created the md files. 

I created the models myself so other than extend laratrustRole

**Reply by PilauGrice** [reply]
wondering if https://spatie.be/docs/laravel-permission/v3/basic-usage/basic-usage 

would work better with it.

**Reply by PilauGrice** [reply]
Hey @ryangjchandler Do relationships work in Orbit ?

**Reply by ryangjchandler** [reply]
Relationships can work yeah. Many to many relationships are sometimes a pain because you need to use a custom pivot model...

**Reply by PilauGrice** [reply]
Its Just a one to many and belongs to but cant seem to get htem to work.

**Comment by PilauGrice** [comment]
HI @ryangjchandler  i did write a few but i didnt manage to get it working do you have an example i can look at for doing this please.
